kolibri/plugins/user/assets/src/views/sign-in-page/index.vue

```diff
@@ -53,10 +53,10 @@
       <p class="login-text no-account">{{ $tr('noAccount') }}</p>
       <div id="btn-group">
         <router-link v-if="canSignUp" class="group-btn" :to="signUp">
-          <icon-button id="signup-button" :text="$tr('createAccount')" :primary="true"/>
+          <k-button :text="$tr('createAccount')" :primary="false"/>
```

Comment:
Could probably remove the 100% width here?

![image](https://user-images.githubusercontent.com/2367265/28465171-af437370-6ddc-11e7-94c5-8e788086c0b9.png)